docs/getting-started/installing-on-mac.rst

```diff
@@ -30,6 +30,8 @@ Once Homebrew is installed, install the DNVM by running ``brew tap aspnet/dnx``
 
 .. image:: installing-on-mac/_static/brew-tap-aspnetdnx.png
 
+If you had an older version of our Homebrew "tap" configured, you might get the following error: ``Could not create link for aspnet/dnx/kvm, as it conflicts with aspnet/k/kvm``. In that case, simply run ``brew uninstall kvm`` to ensure any old versions of ``kvm`` are removed, then ``brew untap aspnet/k`` to remove the old tap. After that, the ``brew tap aspnet/dnx`` command should succeed.
```

Comment:
Make this a note? http://docutils.sourceforge.net/docs/ref/rst/directives.html#note